### ISSUE TITLE
Doesn't work on Android 11

### ISSUE THREAD

**Issue by DonnaTry** [smkhurramkhan/Recover-Deleted-Messages-for-WhatsApp-Facebook#1]
I downloaded and compiled the application. On my Android 10 device it works partially only for images in Whatsapp Images folder. But it doesn't work on Android11.

**Comment by smkhurramkhan** [comment]
yes it will not work because the whatsapp folder is in Android folder in Android 11 you have to change the path of whatsapp folder

**Comment by DonnaTry** [comment]
Yes I found the new location of Whatsapp and changed it. But the application currently only works for the Images folder. How can I run this for both Private and Sent folders at the same time?

**Comment by smkhurramkhan** [comment]
This app recovers images you received in the images folder so that why it works for images folder not private and sent.
